### DISCUSSION TITLE
Connection with SQL Server

### DISCUSSION THREAD

**Discussion by mabeinformatica** [adonisjs/core#4312] [General]
Hi everyone!!!

I'm trying to make a connection with a sql server database, but this error was happing...

{
    "message": "Failed to connect to 192.168.1.249\\SAGEX3:1433 - getaddrinfo ENOTFOUND 192.168.1.249\\SAGEX3",
    "stack": "Error: Failed to connect to 192.168.1.249\\SAGEX3:1433 - getaddrinfo ENOTFOUND 192.168.1.249\\SAGEX3\n    at Connection.socketError (C:\\Users\\acer\\Documents\\Projetos\\Adonis\\x3-api\\node_modules\\tedious\\src\\connection.ts:2309:28)\n    at C:\\Users\\acer\\Documents\\Projetos\\Adonis\\x3-api\\node_modules\\tedious\\src\\connection.ts:2098:37\n    at processTicksAndRejections (node:internal/process/task_queues:77:11)",
    "code": "ESOCKET"
}

I'm sure this connection works, because I use DBeaver and it works, the only difference is that DBeaver uses jdbc connector.

Anyone can help me?

Thanks a lot!!

**Comment by Piero656** [comment]
heeeeeeeeeeeeeelp
